app/src/main/java/de/xikolo/controllers/section/VideoPreviewFragment.kt

```diff
@@ -188,4 +224,52 @@ class VideoPreviewFragment : ViewModelFragment<VideoPreviewViewModel>() {
         }
     }
 
+    private fun displayAvailableSubtitles() {
+        if (video!!.subtitles != null && video!!.subtitles.isNotEmpty()) {
+            textSubtitles.text = VideoDescriptionDelegate
+                .getAvailableSubtitlesText(
+                    getString(R.string.video_settings_subtitles),
+                    video!!.subtitles
+                )
+            textSubtitles.visibility = View.VISIBLE
+        } else {
+            textSubtitles.visibility = View.GONE
+        }
+    }
+
+    private fun displayCollapsedDescription() {
+        textDescription.visibility = View.VISIBLE
+        showDescriptionButton.visibility = View.VISIBLE
+        textDescriptionTitle.visibility = View.VISIBLE
+        textDescription.ellipsize = TextUtils.TruncateAt.MARQUEE
+        textDescription.maxLines = 3
+        showDescriptionButton.text = getString(R.string.show_more)
+
+        if (textDescription.text.length < 115) {
```

Comment:
That's the magic number, right? Actually you want to know if your text was ellipsized. Try out things like that: https://stackoverflow.com/a/4008755/4870048